src/containers/GraphSettingsContainer.tsx

```diff
@@ -0,0 +1,244 @@
+import * as React from 'react';
+import { Button, DropdownButton, Icon, MenuItem, OverlayTrigger, Popover } from 'patternfly-react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { serviceGraphFilterActions } from '../actions/ServiceGraphFilterActions';
+import { KialiAppState, ServiceGraphFilterState } from '../store/Store';
+import { style } from 'typestyle';
+import GraphFilter from '../components/GraphFilter/GraphFilter';
+import { EdgeLabelMode, Layout } from '../types/GraphFilter';
+import { GraphParamsType } from '../types/Graph';
+import { makeServiceGraphUrlFromParams } from '../components/Nav/NavUtils';
+import EdgeLabelRadioGroup from '../components/ToolbarDropdown/EdgeLabelRadioGroup';
+
+interface ServiceGraphDispatch {
+  // Dispatch methods
+  toggleLegend(): void;
+  toggleGraphNodeLabels(): void;
+  toggleGraphCircuitBreakers(): void;
+  toggleGraphVirtualServices(): void;
+  toggleGraphMissingSidecars(): void;
+  toggleTrafficAnimation(): void;
+}
+
+// inherit all of our Reducer state section  and Dispatch methods for redux
+type GraphSettingsProps = ServiceGraphDispatch & ServiceGraphFilterState & GraphParamsType;
+
+// Allow Redux to map sections of our global app state to our props
+const mapStateToProps = (state: KialiAppState) => ({
+  showLegend: state.serviceGraph.filterState.showLegend,
+  showNodeLabels: state.serviceGraph.filterState.showNodeLabels,
+  // edgeLabelMode: state.serviceGraph.filterState.edgeLabelMode,
```

Comment:
Is this disconnected?

Reply:
I tested and `edgeLabelMode` is working. I'm wondering why is this left here commented, can we remove it?

Reply:
Yes, it is disconnected as stated in above comment.

Reply:
I don't want to uncomment because it will get redux connectd in another PR.